### PR TITLE
Clarify documentation for setting the log location

### PR DESCRIPTION
**Pull request by cawallin**

### CODE REVIEW

**Comment by cawallin** [comment]
FYI @KBP-TDC

File: docs/installation/presto-configuration.rst [hunk 1]
```diff
@@ -17,66 +18,83 @@ memory (say, 120GB/node), you may want to allocate more memory to Presto for bet
...
 
-    query.max-memory-per-node=30GB
-    query.max-memory=<30GB * number of nodes>
+        query.max-memory-per-node=30GB
+        query.max-memory=<30GB * number of nodes>
 
 
-We recommend setting ``query.max-memory-per-node`` to half of the JVM config max memory, though if your workload is highly concurrent, you may want
-to use a lower value for ``query.max-memory-per-node``.
+   We recommend setting ``query.max-memory-per-node`` to half of the JVM config max memory, though if your workload is highly concurrent, you may want
```
Comment (akshatnair) [inline docs/installation/presto-configuration.rst]: You may mention that by default it is 10% of the heap space allocated to Presto JVM

File: docs/installation/presto-configuration.rst [hunk 2]
```diff
@@ -17,66 +18,83 @@ memory (say, 120GB/node), you may want to allocate more memory to Presto for bet
...
 
-    node.launcher-log-file=/disk1/presto/launcher.log
-    node.server-log-file=/disk2/presto/server.log
+        node.data-dir=/data1/presto/data
+        node.launcher-log-file=/data1/presto/launcher.log
+        node.server-log-file=/data2/presto/server.log
 
-They can be in the same directory as ``node.data-dir`` if the filesystem has enough space. For these two log files, the
-default location is ``/var/log/presto`` which might not be big enough for a production environment.
+3. The log directory(s) (in the above example, ``/data1/presto`` and ``/data2/presto``; the ``data`` directory
```
Comment (akshatnair) [inline docs/installation/presto-configuration.rst]: s/(s)/(ies)

**Comment by akshatnair** [comment]
lgtm mod comments

**Comment by cawallin** [comment]
updated

**Comment by KBP-TDC** [comment]
@cawallin Would it makes sense to also add `node.launcher-log-file` and `node.server-log-file` to the sample config files we ship with Presto?

**Comment by cawallin** [comment]
Oh yes, you mentioned that too. That's a bit bigger change so I was going to file a ticket for it.

File: docs/installation/presto-configuration.rst [hunk 3]
```diff
@@ -9,74 +10,92 @@ you still may need to make some changes.
...
+        query.max-memory=50GB
 
 
-to ::
+   to ::
 
-    query.max-memory-per-node=30GB
-    query.max-memory=<30GB * number of nodes>
+        query.max-memory-per-node=30GB
+        query.max-memory=<30GB * number of nodes>
```
Comment (maciejgrzybek) [inline docs/installation/presto-configuration.rst]: Shouldn't we refer to data skew? If there's a data skew, `max-memory-per-node` should be higher than `total_memory / number_of_nodes`.

File: docs/installation/presto-configuration.rst [hunk 4]
```diff
@@ -9,74 +10,92 @@ you still may need to make some changes.
...
-2. There are two more log related properties that should be specified with locations that have enough space, these
-are ``node.launcher-log-file`` and ``node.server-log-file``. You can specify them in ``node.properties``: ::
+2. Presto stores logs and other data in ``node.data-dir``, ``node.launcher-log-file``,
+   and ``node.server-log-file``. It is very important that these locations have enough space for the logs on the filesystem on
+   each node where Presto is running. The default location for ``node.data-dir`` is ``/var/lib/presto/data``, the
+   default location for ``node.launcher-log-file`` is ``/var/log/presto/launcher.log``, and the default
+   location for ``node.server-log-file`` is ``/var/log/presto/server.log``.
+   If these locations have limited space, specify a different location for ``node.data-dir``,
+   ``node.launcher-log-file``, and ``node.server-log-file`` in ``/etc/opt/prestoadmin/coordinator/node.properties`` and
+   ``/etc/opt/prestoadmin/workers/node.properties``: ::
```
Comment (maciejgrzybek) [inline docs/installation/presto-configuration.rst]: Maybe add something like:
_Assuming chosen locations are: `/data1` and `/data2` for data directory and server logs respectively, configuration should look as follows:_ ?

File: docs/installation/presto-server-installation.rst [hunk 5]
```diff
@@ -21,8 +21,12 @@ Presto! Presto is now installed on the coordinator and workers specified in your
 The default port for Presto is 8080.  If that port is already in use on your cluster, you will not be able to start Presto.
 In order to change the port that Presto uses, proceed to :ref:`presto-port-configuration-label`.
 
+There are additional configuration properties described at :ref:`presto-configuration-label` that
+must be changed for optimal performance. These configuration changes can be done either
+before or after starting the Presto server and running queries for the first time.
```
Comment (maciejgrzybek) [inline docs/installation/presto-server-installation.rst]: after starting?
Shouldn't we mention it will require restarting server to reflect changes?

Comment (cawallin) [inline docs/installation/presto-server-installation.rst]: I suppose. That page that's linked to very clearly mentions the need to restart the server, but doesn't hurt to add it. Most of this doc work will be thrown away once Brenda finishes, so I do'nt want to spend too much time tweaking it.

**Comment by cawallin** [comment]
updated. also added node.launcher-log-file and node.server-log-file to the sample config files for @KBP-TDC

**Comment by KBP-TDC** [comment]
Looks good. Thanks!

**Pull request merged**